### PR TITLE
Code cleanup and randomness fix for Uncle BF

### PR DESCRIPTION
**Pull request by stropheum**
- Cleaned up code

- Made bracket usage uniform

- Added function prototypes what for viewing main loop more easily

- Added magical confuser pixies what for generating random selections and making the game more high end

- I hope you love me now Uncle Bumblefuck <3

### CODE REVIEW

**Review by Mosnar — APPROVED** [review]

File: main.c [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+const int aH  = 880;
+
+const int speakerPin = 0;	// PWM pin for speaker on the digispark it's pin P0
+const int buttonPin  = 1;	// the number of the button pin P1
+const int rockPin    = 2;	// the number of the ROCK LED pin P2
+const int paperPin   = 3;	// the number of the PAPER LED pin P3
+const int scissorPin = 4;	// the number of the SCISSOR LED pin P4
+
+int buttoncounter = 0;		// integer value for storing how many times the game start reed switch has been actuated
+int counter       = 0;		// integer value for storing the counter for making muzak
```
Comment (Mosnar) [inline main.c:36]: Can this be moved to `void beep()` as to avoid polluting the global scope and cut down memory usage?

Comment (stropheum) [reply main.c:36]: Oh for sure. I just did a rough pass for readability and added randomness just for quality of life type changes. It's a pretty tiny embedded system so I didn't even think to do any efficiency changes

Comment (Mosnar) [reply main.c:36]: Understandable. I just wanted to pretend my computer science degree is worth something for once. Nice work on the PR, this is a huge improvement and beautiful code. @Bumblefuck :shipit:

Comment (stropheum) [reply main.c:36]: Aww my red cheeks =D

**Comment by Bumblefuck** [comment]
Thanks Stropheum! Thanks Mosnar!

**Pull request merged**